shell/src/main/scala/ammonite/shell/ShellSession.scala

```diff
@@ -28,11 +28,11 @@ case class ShellSession() extends OpsAPI {
      */
     val realPath = Option(arg)
       .filter(_.isDir)
-      .orElse(Try(Path(arg.toNIO.toRealPath())).toOption.filter(_.isDir))
+      .orElse(Try(arg.followLinks).toOption.filter(_.isDir))
```

Comment:
The javadoc explains that pretty clearly :)
E.g. if a symlink's target is removed, there will likely be an exception.